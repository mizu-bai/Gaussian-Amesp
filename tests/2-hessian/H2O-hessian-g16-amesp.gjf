%nproc=1
%chk=H2O-hessian-g16-amesp.chk
#p external="python3 -u ../../src/gamesp.py" freq nosymm

H2O hessian task (g16-amesp)

0 1
 O                  0.00000000   -0.00000000   -0.11085125
 H                  0.00000000   -0.78383672    0.44340501
 H                 -0.00000000    0.78383672    0.44340501

