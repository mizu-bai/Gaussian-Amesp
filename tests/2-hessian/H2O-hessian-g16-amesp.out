 Entering Gaussian System, Link 0=g16
 Initial command:
 /opt/g16/l1.exe "/home/jh-li/scratch/g16/Gau-353880.inp" -scrdir="/home/jh-li/scratch/g16/"
 Entering Link 1 = /opt/g16/l1.exe PID=    353883.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                12-Oct-2023 
 ******************************************
 %nproc=1
 Will use up to    1 processors via shared memory.
 %chk=H2O-hessian-g16-amesp.chk
 --------------------------------------------------------
 #p external="python3 -u ../../src/gamesp.py" freq nosymm
 --------------------------------------------------------
 1/10=4,18=10,30=1,38=1/1,3;
 2/12=2,15=1,17=6,18=5,40=1/2;
 3/11=9,25=1,30=1,41=9900000,43=2,71=2,140=1/1;
 4/20=17,22=2,24=3,113=1,114=1/2;
 6/7=2,8=2,9=2,10=2/1;
 7/8=1,25=1,30=1,44=-1/16;
 1/10=4,18=10,30=1/3;
 99//99;
 Leave Link    1 at Thu Oct 12 10:24:20 2023, MaxMem=           0 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l101.exe)
 ----------------------------
 H2O hessian task (g16-amesp)
 ----------------------------
 Symbolic Z-matrix:
 Charge =  0 Multiplicity = 1
 O                     0.        0.       -0.11085 
 H                     0.       -0.78384   0.44341 
 H                     0.        0.78384   0.44341 
 
 ITRead=  0  0  0
 MicOpt= -1 -1 -1
 NAtoms=      3 NQM=        3 NQMF=       0 NMMI=      0 NMMIF=      0
                NMic=       0 NMicF=      0.
                    Isotopes and Nuclear Properties:
 (Nuclear quadrupole moments (NQMom) in fm**2, nuclear magnetic moments (NMagM)
  in nuclear magnetons)

  Atom         1           2           3
 IAtWgt=          16           1           1
 AtmWgt=  15.9949146   1.0078250   1.0078250
 NucSpn=           0           1           1
 AtZEff=  -0.0000000  -0.0000000  -0.0000000
 NQMom=    0.0000000   0.0000000   0.0000000
 NMagM=    0.0000000   2.7928460   2.7928460
 AtZNuc=   8.0000000   1.0000000   1.0000000
 Leave Link  101 at Thu Oct 12 10:24:20 2023, MaxMem=   104857600 cpu:               0.1 elap:               0.1
 (Enter /opt/g16/l103.exe)

 GradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGrad
 Berny optimization.
 Initialization pass.
 Trust Radius=3.00D-01 FncErr=1.00D-07 GrdErr=1.00D-07 EigMax=2.50D+02 EigMin=1.00D-04
 Number of steps in this run=      2 maximum allowed number of steps=      2.
 GradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGrad

 Leave Link  103 at Thu Oct 12 10:24:20 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l202.exe)
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          8           0        0.000000    0.000000   -0.110851
      2          1           0        0.000000   -0.783837    0.443405
      3          1           0        0.000000    0.783837    0.443405
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3
     1  O    0.000000
     2  H    0.960000   0.000000
     3  H    0.960000   1.567673   0.000000
 Symmetry turned off by external request.
 Stoichiometry    H2O
 Framework group  C2V[C2(O),SGV(H2)]
 Deg. of freedom     2
 Full point group                 C2V     NOp   4
 Rotational constants (GHZ):         919.0227249         408.0852063         282.5991538
 Leave Link  202 at Thu Oct 12 10:24:20 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l301.exe)
 Standard basis: STO-3G (5D, 7F)
 Ernie: Thresh=  0.10000D-02 Tol=  0.10000D-05 Strict=F.
     7 basis functions,    21 primitive gaussians,     7 cartesian basis functions
     5 alpha electrons        5 beta electrons
       nuclear repulsion energy         9.1571759236 Hartrees.
 IExCor=    0 DFT=F Ex=HF Corr=None ExCW=0 ScaHFX=  1.000000
 ScaDFX=  1.000000  1.000000  1.000000  1.000000 ScalE2=  1.000000  1.000000
 IRadAn=      5 IRanWt=     -1 IRanGd=            0 ICorTp=0 IEmpDi=  4
 NAtoms=    3 NActive=    3 NUniq=    3 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned off.
 Leave Link  301 at Thu Oct 12 10:24:20 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l402.exe)
 External calculation of energy, first and second derivatives.
 Running external command "python3 -u ../../src/gamesp.py R"
         input file       "/home/jh-li/scratch/g16/Gau-353883.EIn"
         output file      "/home/jh-li/scratch/g16/Gau-353883.EOu"
         message file     "/home/jh-li/scratch/g16/Gau-353883.EMs"
         fchk file        "/home/jh-li/scratch/g16/Gau-353883.EFC"
         mat. el file     "/home/jh-li/scratch/g16/Gau-353883.EUF"
>>> Running Amesp...
>>> Parsing Amesp output...
>>> 大师大法好！
 Recovered energy= -76.4179972776     dipole=     -0.000000000000     -0.000000000000      1.948200000000
 Energy=    -76.4179973     NIter=   0.
 Dipole moment=      -0.000000      -0.000000       1.948200
 Leave Link  402 at Thu Oct 12 10:24:22 2023, MaxMem=   104857600 cpu:               0.0 elap:               1.6
 (Enter /opt/g16/l601.exe)
 Leave Link  601 at Thu Oct 12 10:24:22 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l716.exe)
 Dipole        =-0.00000000D+00-0.00000000D+00 1.94820000D+00
 Full mass-weighted force constant matrix:
 Low frequencies --- -465.1235 -173.2459   -8.4209    3.2456   12.5662  236.7249
 Low frequencies --- 1587.2239 3842.5633 4004.2454
 Diagonal vibrational polarizability:
        0.0000000       0.0000000       0.0000000
 Harmonic frequencies (cm**-1), IR intensities (KM/Mole), Raman scattering
 activities (A**4/AMU), depolarization ratios for plane and unpolarized
 incident light, reduced masses (AMU), force constants (mDyne/A),
 and normal coordinates:
                      1                      2                      3
                      A                      A                      A
 Frequencies --   1587.2239              3842.5633              4004.1232
 Red. masses --      1.0867                 1.0414                 1.0867
 Frc consts  --      1.6131                 9.0597                10.2658
 IR Inten    --      0.0000                 0.0000                 0.0000
  Atom  AN      X      Y      Z        X      Y      Z        X      Y      Z
     1   8    -0.00   0.00  -0.07     0.00   0.00   0.05    -0.00  -0.07  -0.00
     2   1    -0.00   0.41   0.58     0.00   0.60  -0.38    -0.00   0.58  -0.41
     3   1    -0.00  -0.41   0.58     0.00  -0.60  -0.38    -0.00   0.58   0.41

 -------------------
 - Thermochemistry -
 -------------------
 Temperature   298.150 Kelvin.  Pressure   1.00000 Atm.
 Atom     1 has atomic number  8 and mass  15.99491
 Atom     2 has atomic number  1 and mass   1.00783
 Atom     3 has atomic number  1 and mass   1.00783
 Molecular mass:    18.01056 amu.
 Principal axes and moments of inertia in atomic units:
                           1         2         3
     Eigenvalues --     1.96376   4.42246   6.38622
           X           -0.00000   0.00000   1.00000
           Y            1.00000   0.00000   0.00000
           Z            0.00000   1.00000   0.00000
 This molecule is an asymmetric top.
 Rotational symmetry number  2.
 Rotational temperatures (Kelvin)     44.10614    19.58500    13.56262
 Rotational constants (GHZ):         919.02272   408.08521   282.59915
 Zero-point vibrational energy      56427.3 (Joules/Mol)
                                   13.48645 (Kcal/Mol)
 Vibrational temperatures:   2283.66  5528.59  5761.04
          (Kelvin)
 
 Zero-point correction=                           0.021492 (Hartree/Particle)
 Thermal correction to Energy=                    0.024328
 Thermal correction to Enthalpy=                  0.025272
 Thermal correction to Gibbs Free Energy=         0.003876
 Sum of electronic and zero-point Energies=            -76.396505
 Sum of electronic and thermal Energies=               -76.393669
 Sum of electronic and thermal Enthalpies=             -76.392725
 Sum of electronic and thermal Free Energies=          -76.414121
 
                     E (Thermal)             CV                S
                      KCal/Mol        Cal/Mol-Kelvin    Cal/Mol-Kelvin
 Total                   15.266              6.017             45.032
 Electronic               0.000              0.000              0.000
 Translational            0.889              2.981             34.608
 Rotational               0.889              2.981             10.415
 Vibrational             13.489              0.055              0.008
                       Q            Log10(Q)             Ln(Q)
 Total Bot       0.164864D-01         -1.782875         -4.105221
 Total V=0       0.126696D+09          8.102764         18.657303
 Vib (Bot)       0.130187D-09         -9.885434        -22.762052
 Vib (V=0)       0.100047D+01          0.000205          0.000472
 Electronic      0.100000D+01          0.000000          0.000000
 Translational   0.300431D+07          6.477745         14.915559
 Rotational      0.421516D+02          1.624814          3.741273
 -------------------------------------------------------------------
 Center     Atomic                   Forces (Hartrees/Bohr)
 Number     Number              X              Y              Z
 -------------------------------------------------------------------
      1        8          -0.000000000    0.000000000   -0.015384565
      2        1           0.000000000    0.003132839    0.007692282
      3        1           0.000000000   -0.003132839    0.007692282
 -------------------------------------------------------------------
 Cartesian Forces:  Max     0.015384565 RMS     0.006452016
 Z-matrix is all fixed cartesians, so copy forces.
 Force constants in Cartesian coordinates: 
                1             2             3             4             5 
      1 -0.147031D-01
      2  0.000000D+00  0.743799D+00
      3  0.000000D+00  0.000000D+00  0.425822D+00
      4  0.732740D-02  0.000000D+00  0.000000D+00 -0.259505D-02
      5  0.000000D+00 -0.371846D+00  0.203113D+00  0.000000D+00  0.395668D+00
      6  0.000000D+00  0.268136D+00 -0.212907D+00  0.000000D+00 -0.235630D+00
      7  0.732740D-02  0.000000D+00  0.000000D+00 -0.473235D-02  0.000000D+00
      8  0.000000D+00 -0.371846D+00 -0.203113D+00  0.000000D+00 -0.238221D-01
      9  0.000000D+00 -0.268136D+00 -0.212907D+00  0.000000D+00  0.325062D-01
                6             7             8             9 
      6  0.202294D+00
      7  0.000000D+00 -0.259505D-02
      8 -0.325062D-01  0.000000D+00  0.395668D+00
      9  0.106138D-01  0.000000D+00  0.235630D+00  0.202294D+00
 Leave Link  716 at Thu Oct 12 10:24:22 2023, MaxMem=   104857600 cpu:               0.1 elap:               0.1
 (Enter /opt/g16/l103.exe)

 GradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGrad
 Berny optimization.
 Search for a local minimum.
 Step number   1 out of a maximum of    2
 All quantities printed in internal units (Hartrees-Bohrs-Radians)
 Second derivative matrix not updated -- analytic derivatives used.
 The second derivative matrix:
                          X1        Y1        Z1        X2        Y2
           X1          -0.01470
           Y1           0.00000   0.74380
           Z1           0.00000  -0.00000   0.42582
           X2           0.00733  -0.00000   0.00000  -0.00260
           Y2          -0.00000  -0.37185   0.20311   0.00000   0.39567
           Z2           0.00000   0.26814  -0.21291  -0.00000  -0.23563
           X3           0.00733  -0.00000  -0.00000  -0.00473   0.00000
           Y3           0.00000  -0.37185  -0.20311  -0.00000  -0.02382
           Z3           0.00000  -0.26814  -0.21291  -0.00000   0.03251
                          Z2        X3        Y3        Z3
           Z2           0.20229
           X3           0.00000  -0.00260
           Y3          -0.03251   0.00000   0.39567
           Z3           0.01061   0.00000   0.23563   0.20229
 ITU=  0
     Eigenvalues ---    0.14542   0.77044   1.04391
 Angle between quadratic step and forces=  37.33 degrees.
 Linear search not attempted -- first point.
 B after Tr=     0.000000    0.000000   -0.020295
         Rot=    1.000000   -0.000000   -0.000000   -0.000000 Ang=   0.00 deg.
 Variable       Old X    -DE/DX   Delta X   Delta X   Delta X     New X
                                 (Linear)    (Quad)   (Total)
    X1        0.00000  -0.00000   0.00000   0.00000   0.00000   0.00000
    Y1        0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
    Z1       -0.20948  -0.01538   0.00000  -0.03330  -0.05359  -0.26307
    X2        0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
    Y2       -1.48124   0.00313   0.00000   0.04639   0.04639  -1.43484
    Z2        0.83791   0.00769   0.00000   0.04709   0.02680   0.86471
    X3        0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
    Y3        1.48124  -0.00313   0.00000  -0.04639  -0.04639   1.43484
    Z3        0.83791   0.00769   0.00000   0.04709   0.02680   0.86471
         Item               Value     Threshold  Converged?
 Maximum Force            0.015385     0.000450     NO 
 RMS     Force            0.006452     0.000300     NO 
 Maximum Displacement     0.053593     0.001800     NO 
 RMS     Displacement     0.030936     0.001200     NO 
 Predicted change in Energy=-7.637000D-04
 GradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGradGrad

 Leave Link  103 at Thu Oct 12 10:24:22 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l9999.exe)
 Unable to Open any file for archive entry.
 1\1\GINC-DESKTOP-1D9QDE1\Freq\External='python3 -u ../../src/gamesp.py
 '\ZDO\H2O1\JH-LI\12-Oct-2023\0\\#p external="python3 -u ../../src/game
 sp.py" freq nosymm\\H2O hessian task (g16-amesp)\\0,1\O,0.,0.,-0.11085
 125\H,0.,-0.78383672,0.44340501\H,0.,0.78383672,0.44340501\\Version=ES
 64L-G16RevC.01\HF=-76.4179973\RMSD=0.000e+00\RMSF=6.452e-03\ZeroPoint=
 0.021492\Thermal=0.024328\ETot=-76.3936693\HTot=-76.3927251\GTot=-76.4
 141212\Dipole=0.,0.,1.9482\PG=C02V [C2(O1),SGV(H2)]\NImag=0\\-0.014703
 12,0.,0.74379918,0.,0.,0.42582177,0.00732740,0.,0.,-0.00259505,0.,-0.3
 7184576,0.20311253,0.,0.39566778,0.,0.26813639,-0.21290730,0.,-0.23563
 017,0.20229352,0.00732740,0.,0.,-0.00473235,0.,0.,-0.00259505,0.,-0.37
 184576,-0.20311253,0.,-0.02382205,-0.03250619,0.,0.39566778,0.,-0.2681
 3639,-0.21290730,0.,0.03250619,0.01061378,0.,0.23563017,0.20229352\\0.
 ,0.,0.01538456,0.,-0.00313284,-0.00769228,0.,0.00313284,-0.00769228\\\
 @
 The archive entry for this job was punched.


 DON'T WORRY CHARLIE BROWN...WE LEARN MORE FROM LOSING 
 THAN WE DO FROM WINNING.
 THEN THAT MAKES ME THE SMARTEST PERSON IN THE WORLD...
 CHARLES SCHULZ 'PEANUTS'
 Job cpu time:       0 days  0 hours  0 minutes  0.5 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  2.0 seconds.
 File lengths (MBytes):  RWF=      6 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 16 at Thu Oct 12 10:24:22 2023.
